git.lua

```diff
@@ -156,11 +162,17 @@ local function checkout_spec_generator(token)
     --     since it is not added automatically by readline (see previous point)
     clink.matches_are_files(0)
     clink.match_display_filter = function ()
+        local pre = ''
+        local suf = ''
+        if ver.supports_query_rl_var() and rl.isvariabletrue('colored-stats') then
+            pre = color.get_clink_color('color.git.star')
+            suf = color.get_clink_color('color.filtered')
+        end
```

Comment:
Nit: it's shorter to define `star` variable as follows
```suggestion
        local star = '*'
        if ver.supports_query_rl_var() and rl.isvariabletrue('colored-stats') then
            star = color.get_clink_color('color.git.star')..'*'..color.get_clink_color('color.filtered')
        end
```

and use it below
```
         :concat(predicted_branches:map(function(branch) return star..branch end))
```

Reply:
I think reasonable minds can disagree, with good reasons.  I don't want to spend time on that debate.  :)